src/widgets/AutocompleteSelector.js

```diff
@@ -95,11 +97,11 @@ export class AutocompleteSelector extends React.PureComponent {
           autoCapitalize="none"
           autoCorrect={false}
           autoFocus
-          color="white"
-          onChange={text => this.setState({ queryText: text })}
           placeholder={placeholderText}
-          placeholderTextColor="white"
+          onChangeText={this.onChangeText}
           style={[localStyles.text, localStyles.searchBar]}
+          color={WHITE}
+          placeholderTextColor={WHITE}
```

Comment:
`SearchBar` has no `placeholderTextColor` prop. You do not need this line! :)

Reply:
done. Thanks for your time reviewing.